pom.xml

```diff
@@ -26,6 +40,8 @@
         <maven.compiler.source>${java.version}</maven.compiler.source>
         <maven.compiler.target>${java.version}</maven.compiler.target>
 
+        <springCore.version>4.3.23.RELEASE</springCore.version>
```

Comment:
please use the same spring core version as in API Commons

Reply:
Updated.